src/jdk.compiler/share/classes/com/sun/tools/javac/code/Type.java

```diff
@@ -1043,6 +1043,14 @@ public String toString() {
                         buf.append(tsym.packge());
                         buf.append(".");
                     }
+                    ListBuffer<Name> names = new ListBuffer<>();
```

Comment:
There may be a more idiomataic way to javac to construct this string, but offhand I don't know what it is.